warehouse/oidc/forms/_core.py

```diff
@@ -31,13 +31,29 @@ def validate_project_name(self, field):
         project_name = field.data
 
         if project_name in self._project_factory:
+            url_params = {name: value for name, value in self.data.items() if value}
+            url_params["provider"] = {self.provider}
+            url = self._route_url(
+                "manage.project.settings.publishing",
+                project_name=project_name,
+                _query=url_params,
+            )
+
+            # We mark the error message as safe, so that the HTML hyperlink is
+            # not escaped by Jinja
             raise wtforms.validators.ValidationError(
-                _(
-                    "This project already exists, create an ordinary Trusted "
-                    "Publisher instead"
+                markupsafe.Markup(
+                    f"This project already exists, use the project's publishing"
+                    f" settings <a href='{url}'>here</a> to create a Trusted"
+                    f" Publisher for it."
```

Comment:
Shouldn't this be translated?

Reply:
Yeah -- I think we need to either push this down a layer, or make sure the snippet generated here passes through `_(...)` correctly.

Reply:
I'm looking into it now

Reply:
Fixed here: https://github.com/pypi/warehouse/pull/16526